src/components/Popup/EditPopup.js

```diff
@@ -1 +1,102 @@
-import React from 'react';
\ No newline at end of file
+import React from 'react';
+import './editPopup.css';
+
+export default class EditPopup extends React.Component {
+    constructor() {
+        super();
+        this.state = {
+            name: '',
+            surname: '',
+            phone: '',
+            email: '',
+            photo: '',
+            // isOnline: this.props.editData.isOnline
+        };
+    }
+    nameHandler = (e) => {
+        this.setState({ name: e.target.value });
+    };
+    surnameHandler = (e) => {
+        this.setState({ surname: e.target.value });
+    };
+    phoneHandler = (e) => {
+        this.setState({ phone: e.target.value });
+    };
+    emailHandler = (e) => {
+        this.setState({ email: e.target.value });
+    };
+    photolHandler = (e) => {
+        this.setState({ photo: e.target.value });
+    };
+    render() {
+        console.log(this.props)
+        return (
+            <div className="popup">
+                <div className="popup_open">
+                    <div className="box">
+                        <form className="userform">
+                            <div>
+                                <label className="label">Name</label>
+                                <input
+                                    type="text"
+                                    defaultValue={this.props.editData.name}
```

Comment:
if you will use this.props.editData. in multiple places of your code, you can initialize all your properties of this.props.editData in the beginning of your method like this`
const {
  editData: {
    name,
    email,
    ...
  },
  editHandler,
} = this.props;